gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemNewIntegrationTest.java

```diff
@@ -646,6 +646,29 @@ public void listFileInfo_directory() throws Exception {
             bucketUri.resolve(dirObject + "/file1"), bucketUri.resolve(dirObject + "/file2"));
   }
 
+  @Test
+  public void listFileInfo_directory_increased_batch_size() throws Exception {
```

Comment:
This seems to be list API result page size, not batch

Reply:
Ah my bad, didn't realise batch was used in another context. I do mean `pageSize` here